benchopt/cli/main.py

```diff
@@ -158,7 +159,7 @@ def run(benchmark, solver_names, forced_solvers, dataset_names,
     solvers_option = ' '.join(['-s ' + s for s in solver_names])
     forced_solvers_option = ' '.join(['-f ' + s for s in forced_solvers])
     datasets_option = ' '.join(['-d ' + d for d in dataset_names])
-    objective_option = ' '.join(['-p ' + p for p in objective_filters])
+    objective_option = ' '.join(['-p ' + f"'{p}'" for p in objective_filters])
```

Comment:
Why do you need this?

Reply:
It seems that this doesn't work for names that contain blank spaces. For instance, `benchopt run -e -p "Convolutional Sparse Coding[reg=0.1]" benchmark_csc/` returns the following error: `Error: Got unexpected extra arguments (Sparse Coding[reg=0.1])` without adding this fix.